tests/test-proxy.js

```diff
@@ -138,13 +138,13 @@ function addTests() {
       url    : 'https://google.com',
       tunnel : false,
       pool   : false
-    }, true)
+    }, false)
```

Comment:
Because `http_proxy` is only supposed to affect http-requests, not https. Use `all_proxy` to get the original behavior.